lib/setupmanagers/qemuhck/devices/iommu.rb

```diff
@@ -0,0 +1,29 @@
+# typed: true
+# frozen_string_literal: true
+
+require 'sys/cpu'
+
+module AutoHCK
+  class IommuDevice
+    extend T::Sig
+
+    def initialize(logger)
+      @logger = logger
+    end
```

Comment:
I think it is simpler to have a `module` named `QemuHCK::Devices` and have a class method named `iommu`. This removes the need to assign an instance variable.

Reply:
Do you expect to call `device.public_send("iommu")` or have a list of condition `if dev == 'iommu' then device.iommu` ?

Reply:
You can still keep the logic you already have in `read_dynamic_device`. Just replace `AutoHCK.const_get` with `QemuHCK::Devices.public_send`.